src/TLuaInterpreter.cpp

```diff
@@ -9461,6 +9470,15 @@ void TLuaInterpreter::pushMapLabelPropertiesToLua(lua_State* L, const TMapLabel&
     lua_pushstring(L, "Pixmap");
     lua_pushstring(L, label.base64EncodePixmap().constData());
     lua_settable(L, -3);
+    lua_pushstring(L, "OnTop");
```

Comment:
Note that this extends the information returned by `getMapLabel(...)`/`getMapLabel(...)` (to include: whether the map label is on top of everything else; whether it is scaled and whether it is now a temporary one) but it is in a backwards compatible manner... :grinning: